packages/electron-app/src/main/index.ts

```diff
@@ -4,39 +4,31 @@
 
 import { app, BrowserWindow } from 'electron';
 import path from 'path';
-import Pino from 'pino';
 import url from 'url';
 
+import { isSubstrateRunning } from './app/isSubstrateRunning';
 import { initMenu } from './app/menu';
-import { isSubstrateRunning, killSubstrate, runSubstrateDev, staticPath } from './util';
+import { killSubstrate, runSubstrateDev } from './app/substrateProcess';
+// @ts-ignore No idea why we have module not found
+import { productName } from '../../electron-builder.json';
```

Comment:
maybe have to give it an absolute path? It seems to be logging the productName just fine though...

Reply:
Yeah, that's why I put `// @ts-ignore`: it works, just TS is not happy with types.

What do you mean by absolute path?

Reply:
I meant the full path, but having tried it, it makes no difference.